const ThrowError = () => {
    throw new Error("Intentional Error for Phantom Routes");
}
export default ThrowError
